ivy_tests/test_ivy/test_frontends/test_torch/test_non_linear_activation_functions.py

```diff
@@ -104,3 +104,38 @@ def test_torch_softmax(
         dim=axis,
         dtype=dtypes[0],
     )
+
+
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)
+            )
+        )
+    ),
+    as_variable=st.booleans(),
```

Comment:
I found an error here.
AssertionError: Found 1 arrays in the input arguments, but 2 dtypes and as_variable_flags. Make sure to pass in a sequence of bools for all associated boolean flag inputs to test_function, with the sequence length being equal to the number of arrays in the arguments.
So you should not randomly generate the as_variable_flags but set it to False.

Reply:
You should also test for the different possible input for approximate as well and according to the current implementation of ivy.gelu, you should pass False when approximate is 'none' and True in other case. Because ivy.gelu takes boolean.